script/4_PowerSimulationsDynamics_examples/01_omib.jl

```diff
@@ -81,29 +81,32 @@ PSID.execute!(
 
 # ## Exploring the solution
 
+# First, we need to load the simulation results into memory
+results = read_results(sim)
+
 # `PowerSimulationsDynamics` has two functions to obtain different
 # states of the solution:
 #  - `get_state_series(sim, ("generator-102-1", :δ))`: can be used to obtain the solution as
```

Comment:
It should say `get_state_series(results, ("generator-102-1", :δ))` instead of `get_state_series(sim, ("generator-102-1", :δ))`